File: Frontend/etouristagency_mobile/lib/models/passenger/passenger.dart
class Passenger {
  String? id;
  String? fullName;
  String? phoneNumber;
  DateTime? dateOfBirth;

  Passenger(this.id, this.fullName, this.phoneNumber, this.dateOfBirth);

  factory Passenger.fromJson(Map<String, dynamic> json) {
    return Passenger(
      json["id"],
      json["fullName"],
      json["phoneNumber"],
      json["dateOfBirth"] != null ? DateTime.parse(json["dateOfBirth"]) : null,
    );
  }

  Map<String, dynamic> toJson(Passenger passenger){
    return {
      "id" : passenger.id,
      "fullName" : passenger.fullName,
      "phoneNumber" : passenger.phoneNumber,
      "dateOfBirth" : passenger.dateOfBirth
    };
  }
}

File: eTouristAgencyAPI/eTouristAgencyAPI.Services/Database/Models/Passenger.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace eTouristAgencyAPI.Services.Database.Models;

[Table("Passenger")]
public partial class Passenger
{
    [Key]
    public Guid Id { get; set; }

    [StringLength(255)]
    public string FullName { get; set; } = null!;

    [StringLength(50)]
    public string PhoneNumber { get; set; } = null!;

    public Guid ReservationId { get; set; }

    public DateTime DateOfBirth { get; set; }

    public DateTime CreatedOn { get; set; }

    public DateTime ModifiedOn { get; set; }

    public Guid CreatedBy { get; set; }

    public Guid ModifiedBy { get; set; }

    public int? DisplayOrderWithinReservation { get; set; }

    [ForeignKey("CreatedBy")]
    [InverseProperty("PassengerCreatedByNavigations")]
    public virtual User CreatedByNavigation { get; set; } = null!;

    [ForeignKey("ModifiedBy")]
    [InverseProperty("PassengerModifiedByNavigations")]
    public virtual User ModifiedByNavigation { get; set; } = null!;

    [InverseProperty("IdNavigation")]
    public virtual PassengerDocument? PassengerDocument { get; set; }

    [ForeignKey("ReservationId")]
    [InverseProperty("Passengers")]
    public virtual Reservation Reservation { get; set; } = null!;
}
